src/jira/client/jira-client.ts

```diff
@@ -373,6 +428,19 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		security: {
+			submit: async (data, options?: JiraSubmitOptions): Promise<AxiosResponse> => {
```

Comment:
This should probably be `submitVulnerabilities` since we will also have submit workspaces. Not sure what @Harminder84 did in his PR.

Reply:
Doesn't look like either https://github.com/atlassian/github-for-jira/pull/2226/files